src/commandDetails/coin/info.ts

```diff
@@ -15,12 +15,12 @@ const coinInfoExecuteCommand: SapphireMessageExecuteType = async (
 ): Promise<SapphireMessageResponse> => {
   // Get information about coin
   const COIN = getCoinEmoji();
-  const infoEmbed = new MessageEmbed()
+  const infoEmbed = new EmbedBuilder()
```

Comment:
I had to rename all MessageEmbed to EmbedBuilder. The only major change is using addFields with an array. Other methods stayed the same